magento/pwa-studio
# [Doc] fix broken link

## Description

Fixes a broken link in the docs: https://magento.github.io/pwa-studio/technologies/basic-concepts/client-side-caching/

## Related Issue

Closes PWA-773 and PWA-728

Closes #2501 
Closes #2564 

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps


1. Navigate to the docs project: `cd pwa-devdocs`
1. Run the HTML preview server: `yarn develop`
1. Navigate to `/technologies/basic-concepts/client-side-caching/`
1. Verify `src/drivers/adapter.js` link is not broken.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
